azurerm/helpers/validate/strings.go

```diff
@@ -18,3 +18,7 @@ func NoEmptyStrings(i interface{}, k string) ([]string, []error) {
 
 	return nil, nil
 }
+
+func NilOrEmpty(s *string) bool {
```

Comment:
I'm not sure the value of this function? i feel it may be better to simply have the test inline..

Reply:
I agree it's providing borderline value - dropped.

Reply:
Done